sematic/ui/packages/common/src/component/TagsList.tsx

```diff
@@ -29,15 +30,28 @@ const StyledBox = styled(Box)`
 
 interface TagsListProps {
     tags: string[];
+    fold?: number;
     onClick?: (tag: string) => void;
     onAddTag?: () => void;
 }
 
 const TagsList = (props: TagsListProps) => {
```

Comment:
Why is `fold` an input to `TagsList`? Shouldn't `TagsList` determine how to render the raw tags, including truncating them and generating `fold`, based solely on the raw contents of the tags themselves and on the box dimensions?